src/DynamoCore/Migration/Migration.cs

```diff
@@ -1360,6 +1354,7 @@ public class NodeMigrationAttribute : Attribute
         /// <summary>
         /// Version this migrates to.
         /// </summary>
+        [Obsolete("All migrations are now processed, beginning with the first migration whose version is >= the workspace version. The To property will be ignored.", false)]
```

Comment:
I'm not sure how these are used (it's been a while) but I just want to be sure that this change doesn't break them:

https://github.com/DynamoDS/DynamoRevit/blob/7270187e8688fbd8ec538b4617810dccdb9a695f/src/Libraries/RevitNodes/RevitNodes.Migrations.xml